src/turtle_setter.cpp

```diff
@@ -0,0 +1,186 @@
+#include "ros/ros.h"
+#include "turtlesim/Kill.h"
+#include "turtlesim/Spawn.h"
+#include "turtlesim/TeleportAbsolute.h"
+#include "turtlesim/Pose.h"
+#include "software_training_assignment/resetTurtle.h"
+#include "software_training_assignment/distance.h"
+#include "software_training_assignment/SeekTurtleAction.h"
+#include "geometry_msgs/Twist.h"
+#include "actionlib/server/simple_action_server.h"
+#include <cmath>
+
+//This publisher is to be initalized with gemoetery_msgs/Twist !!
+ros::Publisher SeekTurtlePub; //Global since Main needs access to this
+
+
+struct Waypoint
+{
+    float x = 0;
+    float y = 0;
+    float distance = 0;
+    float angle = 0;
+};
+
+static Waypoint movingTurtleWP; //static to keep it file-scope
+
+float distance(float x, float y)
+{
+    return pow(pow(x,2) + pow(y,2), 1/float(2));
+}
+
+float arctan(float x, float y)
```

Comment:
c++ already has an arctan func with cmath, why not use that?

Reply:
The function here calculates the angle from 0 using arctan. I use the cmath function, but this function name is misleading. I will update it :)